module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <array>
+#include <vector>
+
+using pairVector = std::vector<std::pair<uint8_t, uint8_t>>;
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+pairVector compressGrayscale(const std::array<std::array<uint8_t, width>, height>&);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(const pairVector&);
```

Comment:
Have you thought about aliases for longer types like std::array<std::array<uint8_t, width>, height>?